sql/moz-fx-data-shared-prod/fx_quant_user_research_derived/dataset_metadata.yaml

```diff
@@ -13,8 +13,8 @@ workgroup_access:
 - role: roles/bigquery.dataViewer
   members:
   - workgroup:mozilla-confidential
-  - workgroup:fx-quant-user-research
 - role: roles/bigquery.dataEditor
   members:
-  - workgroup:fx-quant-user-research
+  - workgroup:fx-quant-user-research/data-developers
+  - workgroup:fx-quant-user-research/data-viewers
```

Comment:
Why isn't `data-viewers` given the data viewer role?

Reply:
Then wouldn't it make more sense to put them in the developer group instead?  As it is,  adding people to the viewer group won't just give them viewer access which is a potential policy risk